### ISSUE TITLE
Buttons is not working.

### ISSUE THREAD

**Issue by xjunko** [PythonistaGuild/buttons#5]
The code:
```
class buttontest(buttons.Paginator):
    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)

@client.command()
async def test(ctx):
    embed = buttontest(title='embed test', use_defaults=True, entries=['1', '2'], length=1)
    await embed.start(ctx)
```

When button is clicked it will do nothing and then after a few seconds
```
  File "C:\Users\--\AppData\Local\Programs\Python\Python38\lib\site-packages\d
iscord\message.py", line 1032, in remove_reaction
    if member.id == self._state.self_id:
AttributeError: 'NoneType' object has no attribute 'id'
```

This appear.

**Comment by xjunko** [comment]
okay nvm somehow it fixed itself after 2 day

**Comment by Skelmis** [comment]
I am having a similar issue, appears to be system dependant

```
future: <Task finished name='Task-17' coro=<Paginator._session() done, defined at /home/skelmis/Bots/Micros Bot/venv/lib/python3.8/site-packages/discord/ext/buttons/buttons.py:309> exception=AttributeError("'NoneType' object has no attribute 'id'")>
Traceback (most recent call last):
  File "/home/skelmis/Bots/Micros Bot/venv/lib/python3.8/site-packages/discord/ext/buttons/buttons.py", line 320, in _session
    await self._session_loop(ctx)
  File "/home/skelmis/Bots/Micros Bot/venv/lib/python3.8/site-packages/discord/ext/buttons/buttons.py", line 136, in _session_loop
    await self.page.remove_reaction(payload.emoji, ctx.guild.get_member(payload.user_id))
  File "/home/skelmis/Bots/Micros Bot/venv/lib/python3.8/site-packages/discord/message.py", line 1032, in remove_reaction
    if member.id == self._state.self_id:
AttributeError: 'NoneType' object has no attribute 'id'
```

Had to ctrl c to get that since it just is hanging. Will try downgrading d.py and see

**Comment by Skelmis** [comment]
Forced downgrading to the last 1.4.x version of discord.py resolves the issue, however 1.5.x is where the issue appears to lie. This could be caused by something to do with the new introduction of Intents would be my guess, as reaction events are behind them.
